### PR TITLE
Adding ci

### PR DESCRIPTION
**Pull request by camiloribeiro**
@varjoranta Adding travis ci build, code climate and coverage check, also badges to the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,88 +22,6 @@ Exconf follows these basic principles:
...
-```
-exconf template -s hello-world -e local
-```
-
-```
-exconf execute -s hello-world -e local
-```
-
-Continue reading to learn what all of the calls actually mean.
-
```
Comment (varjoranta) [inline README.md:102]: Why did you remove all this? I'm sure it could be made clearer and more concise, but I would like the README to have some usage example and pointing out the environment variable EXCONF_CONFIG_ROOT.

File: README.md [hunk 2]
```diff
@@ -119,12 +41,11 @@ NOTICE: You can change the name of these directories in the *exconf.yaml*, if yo
 NOTICE: Variables "service" and "environment" are set into the variables automatically,
         when you populate the templates or list variables, based on the CLI input.
 
-CLI supports following commands:
+The actions the CLI supports for the configurations:
 * **"execute"** a script, which is usually for deployment purposes for a service. The execution must
   always target some service and some environment defined in the configuration folder.
-* **"template"** command prints out the templates for your service without executing any scripts.
+* **"templates"** command prints out the templates for your service without executing any scripts.
```
Comment (varjoranta) [inline README.md]: The CLI command is actually "template", so not "templates".

**Comment by camiloribeiro** [comment]
@varjoranta Any other considerations?

File: exconf/utils.py [hunk 3]
```diff
@@ -55,8 +55,13 @@ def get_logger(logger_name="magine-services"):
 LOG = get_logger()
 
 
-def read_yaml(file_path):
-    return yaml.load(open(file_path).read())
+def read_yaml(file_path, out=sys.stdout):
+    try:
```
Comment (varjoranta) [inline exconf/utils.py:59]: This will now swallow the exceptions? I think it's better to raise a descriptive error than to just return None in case of error.
